src/views/Beheer/BeheerLayout.vue

```diff
@@ -67,4 +67,22 @@ main {
 :deep(form) {
   max-inline-size: 40rem;
 }
+
+:deep(fieldset) {
```

Comment:
kan dit niet met de algemene styling? formulier styling zou toch vrij uniform moeten kunnen zijn. als het voor nu lastig is dan is dit ok, maar hoe minder van dit, hoe makkelijker toekomstige styling refactoring zal zijn.